tsconfig.json

```diff
@@ -12,8 +12,8 @@
     "experimentalDecorators": true,
     "importHelpers": false,
     "lib": [ "es2017", "esnext" ],
-    "module": "node16",
-    "moduleResolution": "node",
+    "module": "ESNext",
```

Comment:
The unit tests were failing with the error "error TS5110: Option 'module' must be set to 'Node16' when option 'moduleResolution' is set to 'Node16'" even if I've previously set module to node16, so this seems to be a solution for fixing it. Another solution would be to set `isolatedModules` to false, inside `jest.config.ts`. More info can be found here: https://github.com/kulshekhar/ts-jest/issues/4198